kura/maven-central/p2tom2.properties

```diff
@@ -0,0 +1,76 @@
+# mirror.<iu>=false
+# dependency.<ns>.<name>.ignore = true|false
+# mapping.<iu>=groupId:artifactId[:version]
+# external.<iu>=groupId:artifactId[:version]
+
+# Select default mapping strategy
+
+implementation.mavenMapping=org.eclipse.scada.releng.p2.to.maven.DefaultMavenMapping
+scrubJars=true
+
+# Define project metadata
+
+default.project.url=http://eclipse.org/kura
+
+developers=ctron
+
+developer.ctron.name=Jens Reimann
+developer.ctron.email=jreimann@redhat.com
+developer.ctron.url=https://dentrassi.de
+developer.ctron.organization=Red Hat Inc
+developer.ctron.organizationUrl=https://redhat.com
+
```

Comment:
A quick search looks like most projects are not filling in this information. It looks like Jetty and JGit are referencing each committer.

https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-server/9.4.0.v20161208
https://mvnrepository.com/artifact/org.eclipse.jgit/org.eclipse.jgit/4.6.0.201612231935-r